dockstore-webservice/src/main/resources/migrations.1.10.0.xml

```diff
@@ -45,4 +45,7 @@
             SET descriptortype = REPLACE(descriptortype, 'DOCKSTORE_WDL', 'WDL');
         </sql>
     </changeSet>
+    <changeSet author="natalieperez (generated)" id="addNotNullConstraintToTypeColumn">
```

Comment:
Scares me a little if a null value gets introduced, then this migration will fail. I see it's never null in my Jul 24 snapshot, so maybe it's OK. And maybe if we run into this, we should fix it, so yeah, perhaps. Just thinking out loud.

Reply:
If we merge this change, we should make a special note to the wiki instructions for the 1.10 release to make sure we test the migration with a recent DB. 
In some cases we can calculate and populate this value in database, but it might be more trouble than it is worth if it is never null yet.

Reply:
Saw another annotation that makes the fields required, but doesn't need a migration.